.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
   - "0.12"
-  - "4.2"
+  - "4"
```

Comment:
@mroderick you'll probably want 4.3.0 rather than 4.2.0 for now:
https://nodejs.org/en/blog/release/v4.3.0/